phi/tools/eleven_labs_tools.py

```diff
@@ -0,0 +1,73 @@
+"""
+pip install elevenlabs
+"""
+
+from typing import Optional
+from os import getenv, makedirs, path
+from phi.tools import Toolkit
+from phi.utils.log import logger
+from phi.agent import Agent
+from uuid import uuid4
+
+try:
+    from elevenlabs import ElevenLabs  # type: ignore
+except ImportError:
+    raise ImportError("`elevenlabs` not installed. Please install using `pip install elevenlabs`")
+
+
+class ElevenLabsTools(Toolkit):
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        save_path: str = "audio_generations",
+        voice_id: str = "21m00Tcm4TlvDq8ikWAM",
+        model_id: str = "eleven_multilingual_v2",
+        output_format: str = "mp3_44100_64",
```

Comment:
Lets make this a string literal with all the possible options, so it is easier to pick